src/Ninject.Web.AspNetCore/Hosting/AspNetCoreHost.cs

```diff
@@ -23,18 +24,19 @@ public void Start()
 			// * Loading appsettings.json files and environment variables configuration source
 			// * Logging configuration (from appsettings.json)
 			// * AllowedHosts configuration (from appsettings.json)
-			var host = _configuration.WebHostBuilderFactory()
+			var builder = _configuration.WebHostBuilderFactory()
```

Comment:
Just some variable renaming because `host` and `builder` were used very much incorrectly.

Reply:
Yes, that looks much better like this!